modules/update-service-configure-cvo.adoc

```diff
@@ -0,0 +1,39 @@
+[id="update-service-configure-cvo_{context}"]
+
+.Prerequisites
+
+* The OpenShift Update Service Operator has already been installed.
+* The Openshift Update Service graph-data container image has already been created and pushed to a repository accessible to the OpenShift Update Service.
+* The current release and update target releases have already been mirrored to a locally accessible registry.
+* The OpenShift Update Service application has already been created.
+
+.Procedure
+
+. Set the OpenShift Update Service target namespace, for example, `openshift-update-service`:
++
+[source,terminal]
+----
+$ NAMESPACE=openshift-update-service
+----
+
+. Set the name of the OpenShift Update Service application, for example, `service`:
++
+[source,terminal]
+----
+$ NAME=service
+----
+
+.  Obtain the policy engine route:
++
+[source,terminal]
+----
+$ POLICY_ENGINE_GRAPH_URI="$(oc -n "${NAMESPACE}" get -o jsonpath='{.status.policyEngineURI}/api/upgrades_info/v1/graph{"\n"}' updateservice "${NAME}")"
+----
+
+. Patch the CVO ClusterVersion to use the local OpenShift Update Service:
++
+[source,terminal]
+----
+$ PATCH="{\"spec\":{\"upstream\":\"${POLICY_ENGINE_GRAPH_URI}\"}}"
```

Comment:
We only show one command in a code block at a time, can you separate these into separate code blocks/steps?

Reply:
I split these out in the new PR. Hoping I got the added step wording close to right:
```
. Set the patch for the pull graph data:
+
[source,terminal]
----
$ PATCH="{\"spec\":{\"upstream\":\"${POLICY_ENGINE_GRAPH_URI}\"}}"
----
+
. Patch the CVO cluster version to use the local OpenShift Update Service:
+
[source,terminal]
----
$ oc patch clusterversion version -p $PATCH --type merge
----
```